src/store/oauth.ts

```diff
@@ -1,61 +1,38 @@
-import fs from 'fs';
+import {AppContext} from 'mattermost-redux/types/apps';
 
-import {jsonTokenFileStore} from '../utils';
+import {newKVClient, KVClient} from '../clients';
+import {baseUrlFromContext} from '../utils';
 
-interface TokenStore {
-    [key: string]: string;
-}
-
-interface Store {
-    tokens: TokenStore;
+interface AppTokenStore {
     storeToken(userID: string, token: string): void;
     deleteToken(userID: string): void;
-    getToken(userID: string): string;
-    storeTokens(): void;
+    getToken(userID: string): Promise<string>;
 }
 
-class TokenFileStore implements Store {
-    tokens: TokenStore = {};
-
-    constructor() {
-        if (fs.existsSync(jsonTokenFileStore)) {
-            fs.readFile(jsonTokenFileStore, (err, data) => {
-                if (err) {
-                    throw err;
-                }
-                this.tokens = JSON.parse(data.toString());
-            });
-        }
+// need to add prefix
+class TokenStore implements AppTokenStore {
```

Comment:
might me more clear if we follow the `Impl` interface implementation convention across all interface implementations.

Reply:
done.